src/games/strategy/engine/data/GameParser.java

```diff
@@ -55,7 +55,7 @@
   private static final Class<?>[] SETTER_ARGS = {String.class};
   private GameData data;
   private final Collection<SAXParseException> errorsSAX = new ArrayList<SAXParseException>();
-  final static public String dtdFileName = "game.dtd";
+  final static public String DTD_FILE_NAME = "game.dtd";
```

Comment:
Nit, we typically lead with the scope keyword, in this case: "public" , rather than final. This way everything reads as:
`public static final`, or `public static`, or `public final`, or `public`